src/components/Checkbox_Slider/CheckBoxSlider.spec.js

```diff
@@ -0,0 +1,64 @@
+import {render, fireEvent, cleanup} from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import CheckBoxSlide from './CheckBoxSlide';
+
+afterEach(cleanup);
+
+const CHECKBOX_ID = 'my-checkbox';
+const SPAN_ID = 'slider-text';
+
+describe('CheckBoxSlide component', () => {
+	it('should render input', () => {
+		const {getByTestId} = render(<CheckBoxSlide></CheckBoxSlide>);
```

Comment:
If your Component have any children you can use a self closing tag.
```suggestion
		const {getByTestId} = render(<CheckBoxSlide />);
```